README.md

```diff
@@ -26,6 +26,12 @@ Create an HPI file to install in Jenkins (HPI file will be in `target/slack.hpi`
 
     mvn package
 
+If you are having trouble getting mvn to package due to a test failure, please add this to your pom.xml
+
+	<properties>
+		<skipTests>true</skipTests>
+	</properties>
```

Comment:
Test it please then, it's not the change causing the failure, because it occurs without it as well, I'll work on removing the addition to the readme since its such an issue.